Tests/Form/Type/MediaTypeTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Form\Type;
+
+use Sonata\MediaBundle\Form\Type\MediaType;
+use Symfony\Component\Form\Test\TypeTestCase;
+
+/**
+ * @author Christian Gripp <mail@core23.de>
+ */
+class MediaTypeTest extends TypeTestCase
+{
+    /**
+     * @expectedException        \Symfony\Component\OptionsResolver\Exception\MissingOptionsException
+     * @expectedExceptionMessage The required options "context", "provider" are missing.
```

Comment:
You're right, and @Soullivaneuh likes it better too since I showed him [this article](https://thephp.cc/news/2016/02/questioning-phpunit-best-practices)